web/views/cart.hbs

```diff
@@ -0,0 +1,40 @@
+<main class="cart-container">
+  <div class="cart-container--head">
+    <div class="cart-container--head_text">My Cart ({{cart.count}} items)</div>
+  </div>
+
+  {{#each cart.items}}
+  <div class="cart-item" data-id="{{product.id}}">
+    <img src="{{product.imageURL}}" alt="{{product.name}}" class="cart-item__img" />
+    <div class="cart-item-detail">
+      <h1 class="cart-item-detail__heading">{{product.name}}</h1>
+      <div class="cart-item-detail__info">
+        <div class="cart-item-detail__info--count">
+          <a class="cart-item-detail__info--remove" data-id="{{product.id}}">&#45;</a>
+          <span class="cart-item-detail__info--item-count" id=qty-{{product.id}}

```

Comment:
Why its a Span tag used and not a text box? what if i want to change quantity by 20? don't you think clicking a button 20 time is more painful for user?